rs/nns/init/src/lib.rs

```diff
@@ -150,21 +150,26 @@ pub fn make_hsm_sender(hsm_slot: &str, key_id: &str, pin: &str) -> Sender {
 /// NNS canisters, and sets an environment variable pointing at each of them so
 /// that they can be found later.
 pub fn set_up_env_vars_for_all_canisters<P: AsRef<Path>>(wasm_dir: P) {
-    for canister in &NNS_CANISTER_WASMS {
-        // Can it be found?
-        let file_part = format!("{}.wasm", canister);
-        let mut path = wasm_dir.as_ref().to_path_buf();
-        path.push(file_part.as_str());
-        assert!(
-            path.is_file(),
-            "The provided --wasm-dir, '{}', must contain all NNS canister wasms, but it misses {}",
+    'outer: for canister in &NNS_CANISTER_WASMS {
+        // Can either .wasm or .wasm.gz be found?
+        for ext in &[".wasm", ".wasm.gz"] {
```

Comment:
nit: I'd reverse the order here, since wasm.gz is preferable, but otherwise fine

Reply:
Makes sense. Let's do it!